tests/sample/sample.vcxproj

```diff
@@ -192,6 +192,22 @@
       <BuildInParallel>true</BuildInParallel>
     </CustomBuild>
   </ItemGroup>
+  <!-- Build undocked BPF programs that pass verification and build native images for them only when configuration is NOT FuzzerDebug -->
```

Comment:
Can you expand the comment to explain why?

Reply:
Added details in the comment.